api/src/main/java/org/openmrs/api/db/hibernate/HibernateUtil.java

```diff
@@ -30,7 +30,7 @@
 import org.hibernate.SessionFactory;
 import org.hibernate.dialect.Dialect;
 import org.hibernate.dialect.HSQLDialect;
-import org.hibernate.dialect.PostgreSQL82Dialect;
```

Comment:
I still see a number of places in the code base that still reference PostgreSQL82Dialect. 😊